test/fib-dbg.wasm.fromBinary

```diff
@@ -209,6 +209,7 @@
      (br $label$4)
     )
    )
+   ;;@
```

Comment:
I tried to look for this in the binary. What I did is add a printf here:

https://github.com/WebAssembly/binaryen/blob/f44dcd4482f5bbb0fb5b567de4a9425ed949c939/src/wasm/wasm-binary.cpp#L2965

I believe that is the right place for reading a source map entry that says "this has no debug info, explicitly". But I don't see it reached. Am I doing something wrong?

Reply:
No, you are right. One should probably not clear the debug location here:
https://github.com/WebAssembly/binaryen/blob/f44dcd4482f5bbb0fb5b567de4a9425ed949c939/src/wasm/wasm-binary.cpp#L2933